data/specifications.json

```diff
@@ -611,6 +611,12 @@
     "description": "Samsung",
     "MPEG": false
   },
+  {
+    "specification": "OpenAPV",
+    "url": "https://github.com/openapv",
+    "description": "Open Advanced Professional Video Codec",
```

Comment:
The document specifies methods to store data encoded with Advanced Professional Video (APV) codec in ISO Base Media File Format (ISOBMFF) files. APV is a mezzanine video codec for storage, exchange and editing of professional quality video